scripts/validate-schemas.js

```diff
@@ -1,90 +1,60 @@
 const fs = require('fs');
 const path = require('path');
-
 const AjvDraft04 = require('ajv-draft-04');
 const ajvDraft04 = new AjvDraft04();
-
 const Ajv = require('ajv');
 const ajv = new Ajv();
 
-function validation (excludedFiles){
-
-  // Specify the path to the 'schemas' directory
-  const directoryPath = './schemas';
-
-  try{
+function validateSchema(filePath, fileContent, schemaValidator) {
+  try {
+    const obj = JSON.parse(fileContent);
+    const validate = schemaValidator(obj);
 
-    const files = fs.readdirSync(directoryPath);
+    if (validate) {
+      console.log(`\n${filePath}: JSON Schema is valid!`);
+    } else {
+      console.error(`\n${filePath}: JSON Schema is not valid:`, schemaValidator.errors);
```

Comment:
We are already printing errors at the end of the execution. I think we can remove this log line.